drivers/serial/meson/uart.c

```diff
@@ -206,9 +206,9 @@ void init(void)
     uart_setup();
 
 #if !SERIAL_TX_ONLY
-    serial_queue_init(&rx_queue_handle, rx_queue, SERIAL_RX_DATA_REGION_SIZE_DRIV, rx_data);
+    serial_queue_init(&rx_queue_handle, rx_queue, SERIAL_RX_DATA_REGION_CAPACITY_DRIV, rx_data);
 #endif
-    serial_queue_init(&tx_queue_handle, tx_queue, SERIAL_TX_DATA_REGION_SIZE_DRIV, tx_data);
+    serial_queue_init(&tx_queue_handle, tx_queue, SERIAL_TX_DATA_REGION_CAPACITY_DRIV, tx_data);
```

Comment:
This is what I was referring to. I was expecting them to remain as 'size'.

Reply:
Actually this was an interesting case as the serial data region is actually a queue.

Reply:
It's different to the region we use to hold the queues in the networking. I decided to name it capacity, even though at first I thought I'd keep it as size, because the size of the data region in the serial system is actually used as the capacity (e.g. how many bytes can be enqueued in the region).

Reply:
For example, in the the serial queue handle data structure:
```
typedef struct serial_queue_handle {
    serial_queue_t *queue;
    uint32_t capacity;
    char *data_region;
} serial_queue_handle_t;
```
the data region size/capacity is used to initialise the capacity element.